client/components/manageableProduct.js

```diff
@@ -0,0 +1,56 @@
+import React, {Component} from 'react'
+import ProductForm from './productForm'
+import {connect} from 'react-redux'
+import {deleteProductById} from '../store/product'
+
+class ManageableProduct extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      edit: false
+    }
+  }
+
+  handleSelectSubmit = (e, id) => {
+    e.preventDefault()
+    if (e.target.value === 'delete') {
+      this.props.deleteProduct(id)
+      //reroute to AllProducts
+    } else if (e.target.value === 'edit') {
+      this.setState({edit: true})
+    }
+  }
+
+  deleteProduct = id => {
+    this.props.deleteProduct(id)
+  }
+
+  render() {
+    return (
+      <div className="single-admin-product">
+        <div>
+          <h3>{this.props.product.title}</h3>
+          <img src={this.props.product.imageUrl} />
+          <h4>{this.props.product.priceCents}</h4>
+          <h4>{this.props.product.quantity}</h4>
+          <p>{this.props.product.description}</p>
```

Comment:
You're using `this.props.product` a lot here. It would be worth making a variable for that instead.

```js
const { product } = this.props
```